src/components/ResourceForm/AmbulanceFormPage.tsx

```diff
@@ -73,14 +68,17 @@ const AmbulanceFormPage = ({
   });
 
   const [ambulanceNumber, setAmbulanceNumber] = useState<string>('');
+  const [openValidationError, setOpenValidationError] = useState<boolean>(
```

Comment:
nit: `isModalOpen` might be a little clearer and more concise in this context
```suggestion
  const [isModalOpen, setIsModalOpen] = useState<boolean>(
```